### PR TITLE
New package: YggdrasilNetwork.Yggdrasil version 0.5.4

### PR DESCRIPTION
**Pull request by SNMetamorph**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135597)

### CODE REVIEW

**Comment by SNMetamorph** [comment]
@microsoft-github-policy-service agree

File: manifests/y/YggdrasilNetwork/Yggdrasil/0.5.4/YggdrasilNetwork.Yggdrasil.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: YggdrasilNetwork.Yggdrasil
+PackageVersion: 0.5.4
+PackageLocale: en-US
+Publisher: github.com/yggdrasil-network/yggdrasil-go
```
Comment (Trenly) [inline manifests/y/YggdrasilNetwork/Yggdrasil/0.5.4/YggdrasilNetwork.Yggdrasil.locale.en-US.yaml:7]:
```suggestion
Publisher: github.com/yggdrasil-network
```
This should match control panel exactly

File: manifests/y/YggdrasilNetwork/Yggdrasil/0.5.4/YggdrasilNetwork.Yggdrasil.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: YggdrasilNetwork.Yggdrasil
+PackageVersion: 0.5.4
+PackageLocale: en-US
+Publisher: github.com/yggdrasil-network/yggdrasil-go
+PackageName: Yggdrasil Network Service
```
Comment (Trenly) [inline manifests/y/YggdrasilNetwork/Yggdrasil/0.5.4/YggdrasilNetwork.Yggdrasil.locale.en-US.yaml:8]:
```suggestion
PackageName: Yggdrasil Network
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Pull request merged**